battle-service/build.gradle

```diff
@@ -19,12 +19,16 @@ repositories {
 }
 
 dependencies {
+    implementation project(':common')
+    implementation 'org.springframework.boot:spring-boot-starter'
     implementation 'org.springframework.boot:spring-boot-starter-web'
     compileOnly 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
+tasks.register("prepareKotlinBuildScriptModel") {}
```

Comment:
why you need this?

Reply:
i had an error saying "Task 'prepareKotlinBuildScriptModel' not found in project ':battle-service'"

Reply:
> why you need this?

i had an error saying "Task 'prepareKotlinBuildScriptModel' not found in project ':battle-service'"